spec/OpenLineage.json

```diff
@@ -380,7 +383,29 @@
       ],
       "type": "object"
     },
-
+    "OutputStatisticsOutputDatasetFacet": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseFacet"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "rowCount": {
+              "description": "The number of rows written to the dataset",
+              "type": "integer"
+            },
+            "size": {
+              "description": "The size in mb written to the dataset",
```

Comment:
Both the Spark and BigQuery integrations publish this in bytes

Reply:
thanks!